lib/r10k/git/cache.rb

```diff
@@ -16,7 +16,17 @@ class R10K::Git::Cache
 
   include R10K::Settings::Mixin
 
-  def_setting_attr :cache_root, File.expand_path(ENV['HOME'] ? '~/.r10k/git': '/root/.r10k/git')
+  #@api private
+  def self.determine_cache_root
+    if windows?
+      File.join(ENV['LOCALAPPDATA'], 'r10k', 'git')
```

Comment:
Is this intentionally using `r10k` instead of `.r10k`? I don't know a lot about windows, are dotfiles just non-standard there?

Reply:
Do we need to worry at all about `LOCALAPPDATA` not being set?

Reply:
It is intentional to use `r10k` only; the spec test explicitly tests for the absence of that `.` in windows. Dotfiles are not seen as automatically hidden in windows.

I think the %LOCALAPPDATA% question will be better understood when we know exactly what platforms we want to support. But it might be good to be more defensive here, just in case.

Reply:
Looks like we are guaranteed %LOCALAPPDATA%, so I don't think there's any need to be defensive around it.

The really easy to read MS Doc here: https://docs.microsoft.com/en-us/windows/deployment/usmt/usmt-recognized-environment-variables

And the even easier to read wiki page here: https://en.wikipedia.org/wiki/Environment_variable#Windows